rule/var_naming.go

```diff
@@ -22,6 +22,15 @@ var knownNameExceptions = map[string]bool{
 	"kWh":          true,
 }
 
+var meaninglessPackageNames = map[string]struct{}{
```

Comment:
These are _some_ examples of meaningless package names. There are others (like _model_, _base_, _helpers_, ...); should these others also be in the list?

Reply:
I included only those package names that are mentioned in the Go blog and Go wiki. We could list all the meaningless packages in the world, but there are hundreds. Maybe it’s better to add a `blocklistedPackageNames` argument to the rule's config. What do you think?

Reply:
A rule argument with ad-hoc names seems a good idea.

Reply:
Added `badPackageNames` option. Please check.